src/styles/blocks/card__photo.scss

```diff
@@ -0,0 +1,9 @@
+.card__photo {
+  margin: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+  background-position: center;
+  background-size: 100% 100%;
+  background-image: url('./images/imac.jpag');
```

Comment:
There is a typo in the background image path. The file extension should be `.jpg` instead of `.jpag`.